### PR TITLE
list-saved enhancements

### PR DESCRIPTION
**Pull request by Alexjsenn**
For each saved file it now lists the URLs as well as the name (using the same format as list).

### CODE REVIEW

**Comment by Alexjsenn** [comment]
I also realized a test to verify the URLs should be added, which I am working on now, but if I use the same method as the `ListCommandTest.shouldPrintDownloadAndReportURL` test, it will likely fail intermittently (#132)

**Comment by Alexjsenn** [comment]
So I added the test and then ran them a few times, and from the limited iterations I ran, either all tests passed, or both `ListCommandTest.shouldPrintDownloadAndReportURL` and my new test `ListSavedRecordingsCommandTest.shouldPrintURLs` failed together. I guess this makes sense given I wrote the new test in a similar fashion to the old one.

File: src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ListSavedRecordingsCommandTest.java [hunk 1]
```diff
@@ -137,8 +137,46 @@ void shouldExecuteAndPrintSavedRecordings() throws Exception {
...
+        when(exporter.getArchivedDownloadURL(Mockito.anyString())).thenAnswer(
+            new Answer<String>() {
+                @Override
+                public String answer(InvocationOnMock invocation) throws Throwable {
+                    return String.format(
+                            "http://example.com:1234/api/v1/recordings/%s",
+                            invocation.getArguments()[0]);
+                }
+            });
+        when (exporter.getArchivedReportURL(Mockito.anyString())).thenAnswer(
```
Comment (andrewazores) [inline src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ListSavedRecordingsCommandTest.java:156]: There's an extra whitespace after the `when` on this line. Have you done a `mvn spotless:apply` to auto-format the code?

**Comment by andrewazores** [comment]
I'm still really confused about how those tests can intermittently fail, but after all of Victor's investigations I don't feel any closer to discovering what the real root cause might be. Anyway, for the purposes of this PR I'm fine with the tests as implemented, since they mirror the other pre-existing ones for the standard `list` command.

File: src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ListSavedRecordingsCommandTest.java (hunk 1 above)
Comment (Alexjsenn) [reply src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ListSavedRecordingsCommandTest.java]: Fixed!

**Pull request merged**